sched/semaphore/sem_post.c

```diff
@@ -73,10 +73,12 @@ int nxsem_post_slow(FAR sem_t *sem)
 {
   FAR struct tcb_s *stcb = NULL;
   irqstate_t flags;
-  int32_t sem_count;
 #if defined(CONFIG_PRIORITY_INHERITANCE) || defined(CONFIG_PRIORITY_PROTECT)
   uint8_t proto;
 #endif
+  bool sem_blocks = false;
+  const bool mutex = NXSEM_IS_MUTEX(sem);
+  uint32_t mholder = NXSEM_NO_MHOLDER;
```

Comment:
Seems like this store is redundant, it is overwritten unconditionally later ?

Reply:
yes, it is redundant, but some of the compilers in CI give Werror of "possibly uninitialized". This is needed to pass those, and shouldn't harm.

Reply:
Fair enough